src/components/UI/Icons/ArrowDisabledIcon.tsx

```diff
@@ -0,0 +1,24 @@
+import { FC } from "react";
+import { IconProps } from "@/interfaces/IconInterface";
+
+export const ArrowDisabledIcon: FC<IconProps> = ({
+  width = "12",
+  height = "12",
+  color = "#0B0517",
```

Comment:
do we need to define tthis default arrow color in tailwiind.config theme section ?

Reply:
same applied in rest of the cases